src/css/main.scss

```diff
@@ -65,10 +65,6 @@ body {
 
 .dropdown:hover {
   background-color: white;
-
-  .nav-link {
-    color: black !important;
-  }
 }
```

Comment:
Accidentally I found why the `<li>` elements' font color on Menu on React side were becoming black when hovering on dropdown, so I removed it.